packages/pipeline/README.md

```diff
@@ -147,6 +147,20 @@ set the`ZEROEX_DATA_PIPELINE_DB_URL` environment variable to a valid
     starting from that block number.
 7.  Run the migrations and then run your new script locally and verify it works
     as expected.
+8.  After all tests pass and you can run the script locally, open a new PR to
+    the monorepo. Don't merge this yet!
+9.  If you added any new scripts or dependencies between scripts, you will need
+    to make changes to https://github.com/0xProject/0x-pipeline-orchestration
+    and make a separate PR there. Don't merge this yet!
+10. Ask @feuGeneA or @xianny to deploy your changes to the QA environment.
+    Check the [QA Airflow dashboard](http://airflow-qa.0x.org:8080) to make sure
+    everything works correctly in the QA environment.
+11. After code review, merge your PR to 0x-monorepo (and
```

Comment:
Can we suggest code review *before* testing on QA? It's going to be pretty distracting for us to have to deploy everyone's jobs so let's make sure all needed changes are in before doing the QA deploy.